frontend/src/components/carousel/Carousel.jsx

```diff
@@ -0,0 +1,132 @@
+import Slider from "react-slick";
+import "slick-carousel/slick/slick.css";
+import "slick-carousel/slick/slick-theme.css";
+import CarouselCard from "./CarouselCard.jsx";
+
+
+
+const Carousel = () => {
+    const data = [
```

Comment:
Since this test data is repeated everywhere, create a data.js

Reply:
Done